components/SearchFilter.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+interface SearchFilterProps {
+  searchString: string,
+  setSearchString: (searchTerm: string) => void
+}
+
+const SearchFilter = ({ searchString, setSearchString } : SearchFilterProps) => {
+  return (
+    <div className='py-4 flex justify-end items-end'>
+      <input
+        className='rounded-md px-4 py-2 outline-none w-1/3'
```

Comment:
how does this look in mobile?

Reply:
that slipped me ... but
<img width="514" alt="Screen Shot 2022-07-31 at 3 57 20 PM" src="https://user-images.githubusercontent.com/63378463/182048659-ec2373fc-e0c8-4aa5-9697-2d76f8cfc856.png">

Reply:
also seems like we need to put an issue on mobile responsiveness.

Reply:
yea from #87